profile_bluesky/startup/instrument/utils/setup_lakeshores.py

```diff
@@ -0,0 +1,17 @@
+__all__ = ['setup_magnet_lakeshore','setup_diffractometer_lowT']
```

Comment:
Why in `utils` and not in `devices.lakeshores`?

Reply:
These functions actually don't work as I expected, so I will remove them for now.